src/Symfony/Component/Cache/Tests/Adapter/MemcachedAdapterTest.php

```diff
@@ -0,0 +1,277 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Cache\Tests\Adapter;
+
+use Symfony\Component\Cache\Adapter\MemcachedAdapter;
+
+class MemcachedAdapterTest extends AdapterTestCase
+{
+    protected $skippedTests = array(
+        //'testExpiration' => 'Testing expiration slows down the test suite',
+        //'testHasItemReturnsFalseWhenDeferredItemIsExpired' => 'Testing expiration slows down the test suite',
+        //'testDefaultLifeTime' => 'Testing expiration slows down the test suite',
```

Comment:
I assume these should be un-commented prior to any merge of this PR, but in the context of this PR while major changes are continuing to be made, I prefer to disable skipping any tests for this adapter to ensure proper functionality for all the test environments Symfony currently targets with Travis.

**This is a note, though, to not forget to fix this prior to merge!**